### PR TITLE
Updated Tech Program Management 

### LINKED ISSUE #649: Create "Roles In Tech: Technical Program Manager" topic outline & slides
The intention is to present Technical Program Manager roles, and steps to take if one is interested in a career in that field.  Apprentices should walk away knowing more about whether they're a good fit for a future in that work, and how to move forward.

- [ ] Create a topic outline [here](https://github.com/Techtonica/curriculum/blob/master/roles-in-tech/technical-program-manager.md).

- [ ] make a slideshow version of the topic here: https://docs.google.com/presentation/d/1q_sa93_LGQTZhNHyBnQHJmaO_2wgVD_7Mz7-gAjFZ8U/edit?usp=sharing

Acceptance Criteria:
- [ ] The lesson can be covered in 30-45 minutes
- [ ] It is geared towards beginners & junior devs
- [ ] There are clear objectives
- [ ] The activity helps different kinds of learners (e.g. spatial, auditory, verbal, kinesthetic)

### PR DESCRIPTION
**Pull request by imskr**
Fixes #649 and also updated the slides.

### CODE REVIEW

**Comment by imskr** [comment]
@alodahl @manaswinidas Done.

File: roles-in-tech/technical-program-manager.md [hunk 1]
```diff
@@ -2,89 +2,120 @@
...
 
-Apprentices can try to do this other thing. Ideally, they will be challenged to connect what they've learned to some previous knowledge or additional research.
+### Independent Practice
 
+Try to manage or lead a project in your current organization or your team.
 
 ### Check for Understanding
 
-Some ideas: have apprentices summarize to each other, make a cheat sheet, take a quiz, do an assignment, or something else that helps them self-assess their understanding. This exercise should help apprentices determine whether they've met the outline objectives, or if they need to review.
+* Are you able to manage or lead the project.
```
Comment (manaswinidas) [inline roles-in-tech/technical-program-manager.md]: Are these questions? Should be accompanied by question marks

Comment (imskr) [reply roles-in-tech/technical-program-manager.md]: done

File: roles-in-tech/technical-program-manager.md [hunk 2]
```diff
@@ -2,89 +2,120 @@
...
+- Work Responsible for
+- Challenges
 
 ### Materials
 
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
+- [Techtonica Slides](https://docs.google.com/presentation/d/1q_sa93_LGQTZhNHyBnQHJmaO_2wgVD_7Mz7-gAjFZ8U/edit?usp=sharing)
+- [Technical Program Management](http://www.mariogerard.com/technical-program-manager/)
```
Comment (alodahl) [inline roles-in-tech/technical-program-manager.md]: Can you please write a line about how to use this resource?

File: roles-in-tech/technical-program-manager.md [hunk 3]
```diff
@@ -2,89 +2,120 @@
...
+- Career Path
+- Work Responsible for
+- Challenges
 
 ### Materials
 
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
+- [Techtonica Slides](https://docs.google.com/presentation/d/1q_sa93_LGQTZhNHyBnQHJmaO_2wgVD_7Mz7-gAjFZ8U/edit?usp=sharing)
```
Comment (alodahl) [inline roles-in-tech/technical-program-manager.md]: Please see issue #600 on how to format your slides. The content looks perfect!

File: roles-in-tech/security.md [hunk 4]
```diff
@@ -10,20 +10,46 @@
 ### Motivation
```
[the remark this answers is not in the document]
Comment (imskr) [reply roles-in-tech/security.md]: No, I don't know how it got included when I pushed

**Comment by imskr** [comment]
I am closing this. I will open it again as it seems problem in it.

**Comment by alodahl** [comment]
thanks @imskr

**Pull request closed without merging**